src/search_engines.js

```diff
@@ -444,6 +444,79 @@ class GoogleSearch {
     }
     return false;
   }
+
+  changeImageSize(size) {
+    const openTool = document.querySelector(
+        '[class="PNyWAd ZXJQ7c"][jsname="I4bIT"]');
+    if (openTool != null) {
+      openTool.click();
+    }
+    const openSizeDropDown = document.querySelector(
+        '[class="xFo9P r9PaP"][jsname="wLFV5d"][aria-label="Size"]');
+    if (openSizeDropDown != null) {
+      openSizeDropDown.click();
+    }
+    const large = document.querySelector(
+        '[class="MfLWbb"][aria-label="Large"]');
+    const medium = document.querySelector(
+        '[class="MfLWbb"][aria-label="Medium"]');
+    const icon = document.querySelector(
+        '[class="MfLWbb"][aria-label="Icon"]');
+    const dropDownWithSize = document.querySelector(
+        '[class="xFo9P r9PaP Fmo8N"][jsname="wLFV5d"]');
+    if (large != null && medium != null && icon != null) {
+      switch (size) {
+        case 'l':
+          large.click();
+          break;
+        case 'e':
+          medium.click();
+          break;
+        case 'i':
+          icon.click();
+          break;
+        default:
+          break;
+      }
+    } else {
+      // While sized is selected, the dropdown element changes.
+      // Expend dropdown and select the size again.
+      switch (size) {
+        case 'l':
+          if (dropDownWithSize.innerHTML == 'Large') {
```

Comment:
Here and below you are calling dropDownWithSize.innerHTML but dropDownWithSize may be undefined?